pkg/wimutils/iswim.go

```diff
@@ -0,0 +1,30 @@
+package wimutils
+
+import (
+	"bytes"
+	"encoding/binary"
+	"io"
+)
+
+var wimMagic = [8]byte{'M', 'S', 'W', 'I', 'M'}
+
+// IsWIM looks at the first 12 bytes of a file to determine if the underlying stream
+// is a WIM file.
+func IsWIM(r io.ReaderAt) (bool, error) {
```

Comment:
Should there be an iswim_windows.go and iswim_unix.go, where the unix version always returns false/error, as there's no WIM handling on Linux?